backend/core/types/src/backend-swagger.ts

```diff
@@ -4833,6 +4833,59 @@ export interface UnitUpdate {
   id: string
 }
 
+export interface UnitsSummaryUpdate {
```

Comment:
After updating to the latest `main`, can you regenerate this to ensure that this matches the generated structure?